README.rst

```diff
@@ -1,6 +1,130 @@
-==========================
- REANA Demo - Hello World
-==========================
+====================================
+ REANA demo example - "hello world"
+====================================
 
-This repository provides a simple "hello world" example demonstrating data
-analysis encapsulation for running it via the REANA system.
+About
+=====
+
+This repository provides a simple "hello world" application example for `REANA
+<http://reanahub.io/>`_ reusable research data analysis plaftorm.
+
+Code
+====
+
+A simple "hello world" Python code for illustration:
+
+- `helloworld.py <helloworld.py>`_
+
+The program takes several optional arguments such as the name of the person to
```

Comment:
I suggest the following modification:
_The program greets a person (or something else) by printing a greeting character by character to an output file. Several optional arguments such as as the name of the person to greet and a sleeptime period (**[time to wait between each character printout || time to wait between writing characters of greeting]**) can be given:_